keras/layers/wrappers.py

```diff
@@ -227,6 +246,59 @@ def step(x, _):
             y._uses_learning_phase = True
         return y
 
+    def compute_mask(self, inputs, mask=None):
```

Comment:
Please add a docstring describing the overall strategy used to compute the mask.